geps/gep-713/index.md

```diff
@@ -3,1155 +3,1112 @@
-can’t even_ look _at these things._ 😕
+The process of calculating Effective metaresources (Effective policies) consists of walking the hierarchy of target objects, from most specific to least specific (i.e., bottom-up, from the leaves towards the roots of the DAG of target objects) or from least specific to most specific (top-down), map reducing to a single metaresource spec each pair of metaresources adjacent to each other in the hierarchy, applying at each step one of the supported [*merge strategies*](#merge-strategies) (described below), until no more than one spec remains for each effective target.
 
-**Chihiro**: _That's gonna take awhile. Our tooling to show us which policies
-bind to a given workload doesn't go the other direction._
+Between two metaresources in conflict and therefore whose specs are to be merged into one according to a given merge strategy, the least specific metaresource of the pair dictates the merge strategy to apply.
 
-**Ana**: _...wait. You have to_ build tools _to know if retries are turned on??_
+Metaresource kinds that implement more than one merge strategy MUST provide a way in the spec for the instances of the metaresource to specify the chosen strategy that the metaresource controller must use to calculate an effective metaresource out of two instances. If no merge strategy is specified, then implementations SHOULD use more-specific-wins merge strategy by default.
 
-Pause.
+The following subsections define a set of rules to arrange metaresources for conflict resolution, as well as an abstract process to calculate effective metaresources.
 
-**Chihiro**: _Policy attachment is more complex than we’d like, yeah._ 😐
-_Look, how about roll back your `baker` change for now? We can get together in
-the morning and start sorting this out._
+#### Conflict resolution rules
 
-Ana shakes her head and rolls back her edits to the `baker` Deployment, then
-sits looking out over the lake as the deployment progresses.
+If multiple metaresources have the same scope (that is, multiple instances of the same metaresource kind affect the same effective target), this is considered to be a conflict.
 
-**Ana**: _Done. Are things happier now?_
+To determine which metaresources attached to objects in a hierarchy are higher or lower, use the following rules, continuing on ties:
+1. Between two metaresources at different levels of the hierarchy, the one attached higher wins (i.e. dictates the merge strategy to use to resolve the conflict).
+2. Between two metaresources at the same level of the hierarchy, the older metaresource based on creation timestamp wins.
+3. Between two metaresources at the same level of the hierarchy and identical creation timestamps, the metaresource appearing first in alphabetical order by `{namespace}/{name}` wins.
 
-**Chihiro**: _Looks like, thanks. Reckon you can get back to your sailboard._ 🙂
+A metaresource winning over another means this metaresource dictates the merge strategy to apply.
```

Comment:
Please move this above L460. It's important to define what "winning" means before going into the conflict resolution specifics.